excel-connector-product/pom.xml

```diff
@@ -28,4 +28,16 @@
       </plugin>
     </plugins>
   </build>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-deploy-plugin</artifactId>
+          <version>3.0.0-M1</version>
```

Comment:
this should make the build success. the reason is that if the version is not definied it takes the default version (2.7.0) and this can not handle our maven repository url parameter out of github action file